manifests/init.pp

```diff
@@ -157,7 +157,7 @@
       }
     } else { # ! $supported
       notice(sprintf("Your operating system %s is unsupported and will not have \
```

Comment:
TIL: there's sprintf. I wonder if it's really that much value here though since simple string interpolation should also work.

Reply:
I agree.